instruments/AWGs.py

```diff
@@ -16,6 +16,8 @@
 from importlib import import_module
 import os
 import inspect
+from drivers.APS import APS
```

Comment:
if I change awg=APS2(...) I get a similar failure even though the APS2 
driver was registered. I'm not sure what's going on here but I'll dig 
into it.  I'm cc'ing Brian D as well to see if he has an idea why this 
does not work.

Registered Driver APS2
Registered Driver Tek5014
Registered Driver Tek7000
:0: FutureWarning: IPython widgets are experimental and may change in 
the future.
Traceback (most recent call last):
   File "AWGs.py", line 28, in <module>
     awg = APS2(label='BBNAPS1')
NameError: name 'APS2' is not defined


On 9/24/15 3:19 PM, Robert McGurrin wrote:
> Without it you can't run AWGs.py as a standalone for testing. You'll 
> get the error below.
>
> python AWGs.py
> Registered Driver APS2
> Registered Driver Tek5014
> Registered Driver Tek7000
> :0: FutureWarning: IPython widgets are experimental and may change in 
> the future.
> Traceback (most recent call last):
>   File "AWGs.py", line 30, in <module>
>     awg = APS(label='BBNAPS1')
> NameError: name 'APS' is not defined
>
>
> On 9/24/15 3:11 PM, Blake Johnson wrote:
>>
>> In instruments/AWGs.py 
>> <https://github.com/BBN-Q/PyQLab/pull/77#discussion_r40359775>:
>>
>> > @@ -16,6 +16,8 @@
>> >  from importlib import import_module
>> >  import os
>> >  import inspect
>> > +from drivers.APS import APS
>>
>> I suspect that this line is not necessary.
>>
>> —
>> Reply to this email directly or view it on GitHub 
>> <https://github.com/BBN-Q/PyQLab/pull/77/files#r40359775>.
>>
>